docs/en/04_Changelogs/4.0.0.md

```diff
@@ -212,6 +214,104 @@ instead, or if used in an actual XML file use `.CDATA` (see [template casting](/
 
 Where your code once used SQLQuery you should now use SQLSelect in all cases, as this has been removed (check the [3.2.0](3.2.0) upgrading notes).
 
+#### Upgrade code that uses i18n
+
+In many cases, localisation strings which worked in 3.x will continue to work in 4.0, however certain patterns
+have been deprecated and will be removed in 5.0. These include:
+
+ - _t calls with a $context parameter, which is ignored.
```

Comment:
Hmm, this was a useful for translators, since text collector picks it up - we just stopped using it because transifex didn't support it at the time. Looks like support for comments has been added now: https://docs.transifex.com/formats/yaml. I originally considered this a regression when moving from the old getlocalization.com service over to transifex.

Context is fairly essential for good translation quality - for example, an identifier like `CMSMain.HANDLE="handle"` doesn't make it clear to a translator if it's used a verb or noun. The fact that we haven't been including comments is unlikely to generate a lot of noise with translators, but that's mainly because we don't *talk* to translators enough (rather than them not valuing the feature).